server/app/durablejobs/jobs/CopyFileKeyForMultipleFileUpload.java

```diff
@@ -0,0 +1,102 @@
+package durablejobs.jobs;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.annotations.VisibleForTesting;
+import durablejobs.DurableJob;
+import io.ebean.QueryIterator;
+import java.util.Locale;
+import models.ApplicantModel;
+import models.ApplicationModel;
+import models.PersistedDurableJobModel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import repository.AccountRepository;
+import repository.ApplicationRepository;
+import services.applicant.ApplicantData;
+import services.applicant.question.Scalar;
+
+/**
+ * Iterates through all applicants and all their applications and updates them for multiple file
+ * uploads.
+ *
+ * <p>Looks at applicant data for any node containing "file_key" and creates a sibling node
+ * "file_keys" with the same data.
+ */
+public final class CopyFileKeyForMultipleFileUpload extends DurableJob {
+  private static final String FILE_KEY_PROPERTY = Scalar.FILE_KEY.name().toLowerCase(Locale.ROOT);
+  private static final String FILE_KEY_LIST_PROPERTY =
+      Scalar.FILE_KEY_LIST.name().toLowerCase(Locale.ROOT);
```

Comment:
I'm pretty sure I do not want to use display string, because this is the storage/API format (we want the underscore, the display name has a space)

I was super confused that I had to do this though, I would have thought we'd put a method for it on Scalar. So am I missing something?

Reply:
Ahh. Go ahead and leave it how you have it.